docs/fluvio/apis/overview.mdx

```diff
@@ -61,11 +61,12 @@ cluster.
 Once you've got a connection handler, you will want to create a producer for a
 given topic.
 
-The producer could be created with the following configurations: `batch_size`, `compression`, `linger` and `partitioner`.
+The producer could be created with the following configurations: `max_request_size`, `batch_size`, `compression`, `linger` and `partitioner`.
 
 These configurations control the behavior of the producer in the following way:
 
-* `batch_size`: Maximum amount of bytes accumulated by the records before sending the batch. Defaults to 16384 bytes.
+* `max_request_size`: Maximum amount of bytes that the producer can send in a single request. If the record is larger than the max request size, the producer will fail to send the record. Defaults to 1048576 bytes.
+* `batch_size`: Maximum amount of bytes accumulated by the records before sending the batch. If the record is larger than the batch size, the producer will send the record in a single new batch. Defaults to 16384 bytes.
 * `compression`: Compression algorithm used by the producer to compress each batch before sending to the SPU. Supported compression algorithms are `none`, `gzip`, `snappy` and `lz4`.
 * `linger`: Time to wait before sending messages to the server. Defaults to 100 ms.
```

Comment:
Since we're updating this section, can you put batch size and linger together and update linger a little?

`linger`:  Time to wait before sending batches to the server that have not reached maximum batch size

Reply:
Sure, Updated!